src/app/frontend/index.html

```diff
@@ -59,6 +59,7 @@
              ng-if="!$ctrl.isLoginState()">
     <!-- Application content is inserted here. -->
   </kd-chrome>
+
   <!-- build:js static/vendor.js -->
   <!-- bower:js -->
   <!-- Bower JS dependencies are populated here (dev build) and compiled
```

Comment:
Search for `bower` occurrences inside codebase to remove leftovers like this.

Reply:
I did. I must have missed this one comment. `bower:js` comment has to stay this way.